VERSIONING.md

```diff
@@ -18,7 +18,7 @@ changes are:
   reordering parameters, adding a method to an interface or abstract class without adding a default
   implementation.
 
-- ABI changes that could require code using the artifact to be recompiled, but not changed, e.g.,
+- API changes that could require code using the artifact to be recompiled, but not changed, e.g.,
```

Comment:
I think this was intentionally "ABI" as the "Application Binary Interface" or something like that.